### PR TITLE
Add SYNDA drs for CMIP5 and CMIP6 (closes #582)

### LINKED ISSUE #582: Add SYNDA drs
If [synda](http://prodiguer.github.io/synda/) is used in daemon mode, it maintains the directory structure as on ESGF. For CMIP6 this is identical to BADC and DKRZ, but for CMIP5 there is a minor difference in that it leaves out the last directory level and does not separate the files by variable.

It would be nice to add this drs to `config-developer.yml`.

### PR DESCRIPTION
**Pull request by zklaus**
Before you start, please read [CONTRIBUTING.md](https://github.com/ESMValGroup/ESMValCore/blob/master/CONTRIBUTING.md).

**Tasks**

-   [x] [Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do, if you haven't done so already (and add the link at the bottom)
-   [x] This pull request has a descriptive title that can be used in a changelog
~~- Add unit tests~~
-   [x] Public functions should have a numpy-style docstring so they appear properly in the [API documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/api/esmvalcore.html). For all other functions a one line docstring is sufficient.
-   [x] please update the documentation
-   [x] Circle/CI tests pass. Status can be seen below your pull request. If the tests are failing, click the link to find out why.
-   [x] Codacy code quality checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
-   [x] Please use `yamllint` to check that your YAML files do not contain mistakes
-   [x] If you make backward incompatible changes to the recipe format, make a new pull request in the [ESMValTool repository](https://github.com/ESMValGroup/ESMValTool) and add the link below

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #582

### CODE REVIEW

**Comment by zklaus** [comment]
Yes, this is the default structure. Note also that the configuration that you link to above is per selection file. This is not the only, perhaps not even the most common way to download using the daemon.

I'll see about the documentation.

**Comment by valeriupredoi** [comment]
@bouweandela yes - K-man is right since we are using `synda get` in our module which is the basic no-checks command that gets the files noqa :beer:

**Comment by stefsmeets** [comment]
Hi @ESMValGroup/esmvaltool-coreteam , seems like this PR has been sitting for a while 😅 It's a two line fix, can this just be merged?

**Comment by bouweandela** [comment]
No, the documentation has still not been added.

**Comment by stefsmeets** [comment]
> No, the documentation has still not been added.

Thanks @bouweandela , I added a short paragraph to the esmvalcore documentation 👍
https://esmvaltool--583.org.readthedocs.build/projects/ESMValCore/en/583/quickstart/find_data.html#synda

File: doc/quickstart/find_data.rst [hunk 1]
```diff
@@ -75,6 +75,19 @@ first discuss the ``drs`` parameter: as we've seen in the previous section, the
 DRS as a standard is used for both file naming conventions and for directory
 structures.
 
+Synda
+-----
+
+If `synda <http://prodiguer.github.io/synda/>`_ is used to download data,
```
Comment (bouweandela) [inline doc/quickstart/find_data.rst:81]:
```suggestion
If the `synda install <https://prodiguer.github.io/synda/sdt/user_guide.html#synda-install>`_ command is used to download data,
```

**Review by bouweandela — APPROVED** [review]
Thanks @stefsmeets for wrapping this up!

**Pull request merged**